src/avram/where.cr

```diff
@@ -245,71 +245,119 @@ module Avram::Where
     end
   end
 
-  class JSONHasKey < ValueHoldingSqlClause
+  class JsonbHasKey < ValueHoldingSqlClause
     def operator : String
       "?"
     end
 
-    def negated : NotJSONHasKey
-      NotJSONHasKey.new(column, value)
+    def negated : NotJsonbHasKey
+      NotJsonbHasKey.new(column, value)
     end
   end
 
-  class NotJSONHasKey < ValueHoldingSqlClause
+  class NotJsonbHasKey < ValueHoldingSqlClause
     def operator : String
       "?"
     end
 
-    def negated : JSONHasKey
-      JSONHasKey.new(column, value)
+    def negated : JsonbHasKey
+      JsonbHasKey.new(column, value)
     end
 
     def prepare(placeholder_supplier : Proc(String)) : String
       "NOT(#{column} #{operator} #{placeholder_supplier.call})"
     end
   end
 
-  class JSONHasAnyKeys < ValueHoldingSqlClause
+  class JsonbHasAnyKeys < ValueHoldingSqlClause
     def operator : String
       "?|"
     end
 
-    def negated : NotJSONHasAnyKeys
-      NotJSONHasAnyKeys.new(column, value)
+    def negated : NotJsonbHasAnyKeys
+      NotJsonbHasAnyKeys.new(column, value)
     end
   end
 
-  class NotJSONHasAnyKeys < ValueHoldingSqlClause
+  class NotJsonbHasAnyKeys < ValueHoldingSqlClause
     def operator : String
       "?|"
     end
 
-    def negated : JSONHasAnyKeys
-      JSONHasAnyKeys.new(column, value)
+    def negated : JsonbHasAnyKeys
+      JsonbHasAnyKeys.new(column, value)
     end
 
     def prepare(placeholder_supplier : Proc(String)) : String
       "NOT(#{column} #{operator} #{placeholder_supplier.call})"
     end
   end
 
-  class JSONHasAllKeys < ValueHoldingSqlClause
+  class JsonbHasAllKeys < ValueHoldingSqlClause
     def operator : String
       "?&"
     end
 
-    def negated : NotJSONHasAllKeys
-      NotJSONHasAllKeys.new(column, value)
+    def negated : NotJsonbHasAllKeys
+      NotJsonbHasAllKeys.new(column, value)
     end
   end
 
-  class NotJSONHasAllKeys < ValueHoldingSqlClause
+  class NotJsonbHasAllKeys < ValueHoldingSqlClause
     def operator : String
       "?&"
     end
 
-    def negated : JSONHasAllKeys
-      JSONHasAllKeys.new(column, value)
+    def negated : JsonbHasAllKeys
+      JsonbHasAllKeys.new(column, value)
+    end
+
+    def prepare(placeholder_supplier : Proc(String)) : String
+      "NOT(#{column} #{operator} #{placeholder_supplier.call})"
+    end
+  end
+
+  class JsonbIncludes < ValueHoldingSqlClause
+    def operator : String
+      "@>"
+    end
```

Comment:
One thing I was just thinking about... This operator can be used on Arrays too. [Ref](https://github.com/luckyframework/avram/issues/968)... If this just became the new `Includes` then it would work for both Arrays and Jsonb, but that also means that the old Includes using `val = ANY(col)` would not be an option without some sort of flag or rename or something...

Reply:
If they're equivalent operators, I see no reason not to simplify.